### PR TITLE
update the position of the tooltip and set it near the label text

### LINKED ISSUE #12643: Bug report: poor tooltip position - find wallet
### Describe the bug

in some instances the tooltip is too far from the text
Screenshot 2024-04-03 at 11 55 09

### To reproduce

1. Go to https://ethereum.org/en/wallets/find-wallet/
2. Click on wallet detail

### Expected behavior

the tooltip should be right after the text without blank gap

### Screenshots

_No response_

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional context

_No response_

### Would you like to work on this issue?

- [ ] Yes
- [X] No

### PR DESCRIPTION
**Pull request by Dharmik79**
## Description

Added the tooltip inside the feature label text and made the list item aligned horizontally

## Preview URL

https://deploy-preview-12779--ethereumorg.netlify.app/en/wallets/find-wallet

## Related issue
- Fixes #12643

### CODE REVIEW

File: src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx [hunk 1]
```diff
@@ -86,21 +87,20 @@ export const WalletMoreInfoCategory = ({
               >
                 <ListIcon as={FeatureIcon}></ListIcon>
 
-                <Text px={1} lineHeight={1}>
+                <Text as="span" whiteSpace="nowrap" px={1} lineHeight={1}>
```
Comment (wackerow) [inline src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx:90]: I don't think we want to use `nowrap` here...

image

It causes overflow.

This was why I had suggested breaking out the last word and attaching the icon to just that word... so the icon will never wrap by itself, but the words still will. Unfortunately there is no CSS pseudo-selector for ":last-word", which would make this easier. 

Hm.. thinking more here though, perhaps we could use a non-breaking-space `&nbsp;` between the string and the Emoji instead? Haven't tried yet, but could be worth a shot... should prevent the emoji from wrapping alone while allowing the spaces between the words to break a line.

**Review by wackerow — CHANGES_REQUESTED** [review]

File: src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx (hunk 1 above)
Comment (wackerow) [reply src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx:90]: I think using `&nbsp;` ("non-breaking space") would work actually, @Dharmik79 are you down to give this a try?

Comment (Dharmik79) [reply src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx]: @wackerow The solution using &nbsp worked.

File: src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx [hunk 2]
```diff
@@ -87,20 +88,19 @@ export const WalletMoreInfoCategory = ({
                 <ListIcon as={FeatureIcon}></ListIcon>
 
                 <Text px={1} lineHeight={1}>
-                  {feature.label}
+                  {feature.label}&nbsp;
```
Comment (wackerow) [inline src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx:91]: ☹️ This is pesky... I'm still seeing this wrap. Made some adjustments locally but only getting it to work on Safari but not Brave.
In either case, I would stick with either the non-breaking space, or the margin (`ms="2"` below) but not both... I'm still not sure of a clean way to do this for all browsers.

Comment (wackerow) [reply src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx:91]: @Dharmik79 Just checking in... you still willing/able to help with this, or would you prefer someone to try and take it from here?

Comment (Dharmik79) [reply src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx:91]: @wackerow I think removing the ms="2" will be a better option.

**Comment by wackerow** [comment]
Circling through, still seeing buggy positioning... going to switch this to draft mode.

**Comment by corwintines** [comment]
Just to confirm what you mean by buggy positioning @wackerow, is it because the Tooltip icon is aligning with the top of the text, and not center?

**Comment by wackerow** [comment]
@corwintines I'm seeing this

image

..with the info icon on it's own line

**Review by corwintines — APPROVED** [review]
LGTM, thanks @Dharmik79!

File: src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx [hunk 3]
```diff
@@ -66,6 +66,7 @@ export const WalletMoreInfoCategory = ({
                 listStyleType="none"
                 display="flex"
                 mb={2}
+                alignItems="center"
```
Comment (wackerow) [inline src/components/FindWallet/WalletTable/WalletMoreInfoCategory.tsx:69]: image

We should revert this as it's out of scope with issue #12643, which was only referring to the positioning of the info "i" icon.

```suggestion
```

If this is a change you'd like to suggest please open a new issue for it.

**Review by wackerow — CHANGES_REQUESTED** [review]
Going to revert the alignment adjustment for the containing flex box, then pull this in

**Review by wackerow — APPROVED** [review]
Okay, LGTM! Thanks @Dharmik79

**Pull request merged**